internal/service/organizations/account.go

```diff
@@ -140,6 +139,7 @@ func resourceAccountCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	d.SetId(aws.StringValue(output.AccountId))
+	d.Set("govcloud_id", output.GovCloudAccountId)
```

Comment:
Yes, this works well. The only limitation is that `govcloud_id` will not be available if a an existing account is imported. At the the moment I don't think that is a blocker.

Reply:
Yes I can also confirm, creating an commercial account, refreshing the status and deleting it (in terraform with `close_on_deletion = true`, still waiting for the closing of the account on AWS side) works for me.

Reply:
Should I maybe add a comment that this line is untested? As no one of us has a govcloud account?

Reply:
I think that is unnecessary since we'll have to take it out after we hear back. New features, especially niche ones, may not be 100%.

Reply:
But we shouldn't break existing functionality 😺.